src/librustc_trans/save/dump_csv.rs

```diff
@@ -70,6 +72,9 @@ pub struct DumpCsvVisitor<'l, 'tcx: 'l> {
     fmt: FmtStrs<'l, 'tcx>,
 
     cur_scope: NodeId,
+
+    // Map of macro definition spans to the set of (known) callsite spans.
```

Comment:
Could you document why this exists here? I.e., that we only want to write each def and use once?

Rather than have a mapping, could we just have two hash sets - one of defs and one of uses? (Do we ever have two uses with the same span but different defs? (I don't think so, but...)).